command.go

```diff
@@ -922,15 +922,14 @@ func (c *Command) DebugFlags() {
 
 // Name returns the command's name: the first word in the use line.
 func (c *Command) Name() string {
-	if c.name != "" {
```

Comment:
I'm good with the whole PR until here. Why do the entire work of the function inside a conditional? I much prefer early returns to blocks of code inside conditionals. But you are more active than me these days, if you strongly prefer this style fix I won't stand in your way.

Reply:
@eparis This change is breaking [Kubernetes](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53631/pull-kubernetes-bazel-test/9403/).

The `c.name` will not be override in [deprecatedAlias](https://github.com/kubernetes/kubernetes/blob/e57accbd03b1265340fdd266d0afe2b7a8e1c171/pkg/kubectl/cmd/cmd.go#L401-L414), since `c.name` will be assigned a value at the first calling and keeps unchanged. You can have a test using [Test_deprecatedAlias](https://github.com/kubernetes/kubernetes/blob/e57accbd03b1265340fdd266d0afe2b7a8e1c171/pkg/kubectl/cmd/cmd_test.go#L672-L720)

Reply:
But I don't see any difference between the functionality of these two changes. In old version this function works the same: `c.name will be assigned a value at the first calling and keeps unchanged`.

Reply:
I propose to delete `name` property of command and generate name on each call of Name() func.

Reply:
> But I don't see any difference between the functionality of these two changes. In old version this function works the same: `c.name will be assigned a value at the first calling and keeps unchanged`.

@bogem Right. I found the root cause is #390, another PR that made by me 😂.

> I propose to delete name property of command and generate name on each call of Name() func.

+1, since `name` is a private attribute and only used in this method. Seems useless. @eparis WDYT?